src/SharedFrameworkValidation/ApiCompatBaseline.LKGRuntime.CurrentRuntime.txt

```diff
@@ -0,0 +1,875 @@
+Compat issues with assembly Microsoft.CodeAnalysis:
```

Comment:
We need a list of baselines somewhere, but I'm OK having it in two places to avoid the noise.